src/main/java/com/strandls/document/es/util/DocumentIndex.java

```diff
@@ -0,0 +1,35 @@
+package com.strandls.document.es.util;
+
+public enum DocumentIndex {
+
+	index("document"),
```

Comment:
put all the contants in upper case letter, constants and enums are always uppercase litreals